code/modules/mob/living/carbon/carbon.dm

```diff
@@ -617,6 +617,9 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 		throw_mode_off()
 	else
 		throw_mode_on()
+	var/obj/item/I = get_active_hand()
+	if(I)
```

Comment:
What happens if it ain't in your active hand? It prob won't start spinning right?

Reply:
yep. thats intended